Corona-Warn-App/src/main/res/layout/item_person_overview.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Would prefer naming by feature, e.g. `person_overview_item.xml`

Reply:
If we group by the whole feature, maybe `covidcertificate_person_overview_item.xml`? Like packagename?